src/Native/LdaNative/CMakeLists.txt

```diff
@@ -16,4 +16,18 @@ set(SOURCES
 
 add_library(LdaNative SHARED ${SOURCES} ${RESOURCES})
 
+if (${ARCHITECTURE} STREQUAL "arm")
+    if (WIN32)
+        target_link_libraries(LdaNative shell32.lib)
+    else()
+        target_link_libraries(LdaNative atomic.a)
+    endif()
+    
+    # Arcade is adding mthumbs and it sets floats to 16 bits. Need to remove it.
```

Comment:
Is there a change needed in Arcade? Or is this the right approach here?

Reply:
I had assumed that since it was in the Arcade code that it was being used by something else.

I can reach out and see if anyone knows why that flag is there.

Reply:
Did we get a resolution here?

Reply:
Not yet. The person that I know to talk to, Aaron Robinson, is currently OOF. Do you know of others who are familiar with that part of arcade? If not I'll just wait till he gets back.

Reply:
So Jan Vorlicek said we shouldn't need to remove it, but has confirmed that we have an issue when its present. He is investigating further to see where the problem is.

So I guess currently we need to remove it for our code to work until his investigation is done. It may be an issue in the runtime itself, so we will probably need to keep it removed is my guess.

Reply:
Alright, he figured it out. Turns out if the `-mthumbs` flag is added, you also need to add `-fPIC` and `-march=armv7-a` for it to work correctly. Without those 2 flags it does compile for 16 bits. The runtime is manually adding those flags in their cmake files, and they are not included in the cross toolchain that arcade provides. I have added those 2 flags and removed my code that removed the `-mthumbs` flag. I am not sure why arcade is adding a flag that by itself when it alone causes problems. I suggested to Jan that they either remove the `-mthumbs` flag from the cross toolchain or they add the other 2 that are needed. Either way just so that the default configuration actually works, instead of being broken.

Reply:
😄 Jan V was going to be my suggestion. Can you open a bug in dotnet/arcade for this?

Reply:
Bug has been opened here https://github.com/dotnet/arcade/issues/7607